Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG golang_image=public.ecr.aws/eks-distro-build-tooling/golang:1.21
+ARG golang_image=public.ecr.aws/eks-distro-build-tooling/golang:1.22
```

Comment:
This is not necessary actually, because the ARG can be overridden

Reply:
Thanks! @xdu31. Should I change it back? I have changed references in all places to maintain consistency.
Would it be possible to get an approval?

Reply:
I think its better to change it here instead of overriding when possible.

Reply:
Resolving this thread.